akka-cluster-bootstrap/src/main/resources/rp.conf

```diff
@@ -1,23 +1,27 @@
-# This needs to be in application.conf because reference.conf files have no guarantee
-# and Akka's values may override ours. If we were strictly in a Play app, we could use
-# reference-overrides.conf but as Akka Cluster also can be used standalone, here we are.
-
 akka {
   actor {
     provider = cluster
   }
+  
+  cluster {
+    bootstrap {
+      contact-point-discovery {
+        discovery-method = akka.discovery.reactive-lib
+      }
+    }
+  }
 
-  extensions += "com.lightbend.rp.akkaclusterbootstrap.ClusterBootstrap"
+  discovery {
+    reactive-lib {
+      class = com.lightbend.rp.akkaclusterbootstrap.ClusterServiceDiscovery
 
-  remote {
-    netty.tcp {
-      hostname = ${?RP_ENDPOINT_AKKA_REMOTE_HOST}
-      port = ${?RP_ENDPOINT_AKKA_REMOTE_HOST}
+      resolve-interval = 1 second
 
-      bind-hostname = ${?RP_ENDPOINT_AKKA_REMOTE_BIND_HOST}
-      bind-port = ${?RP_ENDPOINT_AKKA_REMOTE_BIND_HOST}
+      resolve-timeout = 2 second
     }
   }
+
+  extensions += "com.lightbend.rp.akkaclusterbootstrap.ClusterBootstrap"
```

Comment:
Perhaps this works when defined in application.conf (rp.conf included in application.conf). Anyway, we had trouble with that when trying to auto enable Akka Diagnostics from reference.conf and therefore `akka.library-extensions` was added:
https://github.com/lightbend/akka-commercial-addons/blob/master/akka-diagnostics/src/main/resources/reference.conf#L11